appveyor.yml

```diff
@@ -0,0 +1,30 @@
+# appveyor file
+# http://www.appveyor.com/docs/appveyor-yml
+
+init:
+  - git config --global core.autocrlf input
+
+# what combinations to test
+environment:
+  matrix:
+    #node.js
+    - nodejs_version: 0.10
+    - nodejs_version: 0.11
+    - nodejs_version: 0.12
+    #io.js
+    - nodejs_version: 2.5.0
```

Comment:
@mkorakakis Yeah that's fine. We'll try to add back once iojs is more stable and more dependencies support it.